src/App.jsx

```diff
@@ -1,35 +1,25 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useState } from "react";
+import "./App.css";
+import { Routes, Route } from "react-router-dom";
+import Header from "./components/Header";
+import LandingPage from "./components/LandingPage";
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [articles, setArticles] = useState([]);
```

Comment:
is this the best place to declare the articles if they are being fetched within the articles component? We shouldn't need to use these articles elsewhere so the state should be declared within your landing page component